ios/FluentUI.Demo/FluentUI.Demo/Demos/ObjectiveCDemoController.m

```diff
@@ -97,16 +101,10 @@ - (void)viewDidLoad {
         [self showAlertForCellTapped:@"Sample Title3"];
     }];
 
-    MSFListSectionState *section = [[MSFListSectionState alloc] init];
-    [section setCells:@[listCell1, listCell2, listCell3]];
-    NSArray *sections = @[section];
-
-    MSFList *list = [[MSFList alloc] initWithSections:sections];
-
-    UIView *listView = [list view];
+    UIView *listView = [_testList view];
     listView.translatesAutoresizingMaskIntoConstraints = false;
 
-    [self.container addArrangedSubview:[list view]];
+    [self.container addArrangedSubview:[_testList view]];
```

Comment:
Shouldn't this use listView directly?

`[self.container addArrangedSubview:listView];`